src/main/java/uk/gov/hmcts/ccd/endpoint/exception/RestExceptionHandler.java

```diff
@@ -40,4 +42,13 @@ public RestExceptionHandler(AppInsights appInsights) {
         LOG.warn("exception with message {}, {}, {}", errorMsg, HttpStatus.INTERNAL_SERVER_ERROR, request, e);
         return handleExceptionInternal(e, errorMsg, new HttpHeaders(), HttpStatus.INTERNAL_SERVER_ERROR, request);
     }
+
+    @ExceptionHandler(SQLException.class)
+    protected ResponseEntity<Object> handleSQLException(final SQLException e, final WebRequest request) {
+        final String errorMsg = "SQL Exception thrown during API operation, " + HttpStatus.INTERNAL_SERVER_ERROR;
+        appInsights.trackException(e);
+
+        LOG.warn(errorMsg);
+        return handleExceptionInternal(e, errorMsg, new HttpHeaders(), HttpStatus.INTERNAL_SERVER_ERROR, request);
```

Comment:
Error message should be returned inside a json object. All responses should be json.